socialapp/src/components/Posts/PostCard.jsx

```diff
@@ -188,40 +193,62 @@ export default function PostCard(props) {
   }, []);
 
   const sharePost = (post) => {
-    console.log(post.id);
+    host = baseURL + '/'
     if (post.visibility === "PUBLIC") {
       for (let index = 0; index < followers.length; index++) {
-        const sharedPost = {
-          type: "post",
-          summary: `${author.displayName} shared a post.`,
-          author: author,
-          object: post.id,
-        };
-        api
-          .post(`${baseURL}/authors/${extractAuthorUUID(followers[index].id)}/inbox/`, post)
-          .then((response) => {
-            console.log(response);
-          })
-          .catch((error) => {
-            console.log("Failed to get posts of author. " + error);
+        author = followers[index]
```

Comment:
`author` is ambiguous with the existing `author` state variable
```suggestion
        follower = followers[index]
```